contents/newsletter/remote-working.md

```diff
@@ -0,0 +1,101 @@
+
+If you’re a company built around [big](https://www.cnbc.com/2023/06/08/google-to-crack-down-on-hybrid-work-asks-remote-workers-to-reconsider.html), [shiny](https://gizmodo.com/apple-remote-work-return-to-office-hybrid-work-1850256210) [offices](https://www.cnn.com/2022/06/01/tech/elon-musk-tesla-ends-work-from-home/index.html), then chances are you have a two-tiered system. Essentially, an employee’s ability to move up in such a company is closely linked to their proximity to the biggest, shiniest office. Dell’s new policy acknowledges this outright: if you’re a remote worker at Dell, there’s no career progression. The same is true at many companies that have a mix of remote, hybrid, and office workers.
+
+### The solution? Aggressive transparency
+
+Rather than presentism and nepotism, career progression should be linked to a transparent framework of measurable goals. Do the work, do it well, deliver value for customers, and you’ll be rewarded. Leif Singer, a software engineer at Automattic, the company behind WordPress, has shared some great examples of [how this works in practice](https://youtu.be/aRiP72kRrnc?feature=shared). 
+
+A two-tiered system is built on the bizarre notion that presenteeism is how you get ahead. Clearly defining and then publishing your system for career progression is an important first step to making sure that everyone, regardless of location, is assessed using the same objective criteria. At PostHog, the way we approach [career progression](https://posthog.com/handbook/people/career-progression) and [feedback](https://posthog.com/handbook/people/feedback) is laid out in public for anyone to see.
+
+But transparency doesn’t stop there. Remote work pioneer GitLab does everything out in the open and has published a [2,000-page handbook](https://handbook.gitlab.com/handbook/values/) covering, well, everything. At PostHog, [our handbook](https://posthog.com/handbook) is one of the most important documents we’ve ever written. Why? If you want to find out anything about the company, you can – from what [individual teams](https://posthog.com/teams) are working on right now, to how much we would pay to [hire a full stack engineer in Brunei](https://posthog.com/handbook/people/compensation), to how our co-founders [like](https://posthog.com/handbook/company/team/james-hawkins) to [work](https://posthog.com/handbook/company/team/tim-glaser). This transparent approach gives everyone access to the same information and allows them to focus on doing their best work, not navigating layers of corporate bureaucracy.
+
+### The problem? Communication is messy
+
+Making remote work actually work requires making big cultural and procedural changes. Fixing how your company communicates is one of the hardest as it requires everyone to buy in and change their behavior. That means saying goodbye to email, setting fire to Microsoft Teams, and training people to be much more effective communicators. Here’s how.
```

Comment:
Just generally, I think we need change the language to more about "building" a successful remote company, rather than fixing an existing one. Our audience is founders / aspiring founders building their own companies, not existing execs trying to fix their bad systems.

TL;DR: It's "avoid these mistakes" not "how to fix your mistakes"

Reply:
Can you give a few specific examples so I can zero in on where the language is wrong? Thanks!

Reply:
Some examples:

> "If you’re a company built around [big](https://www.cnbc.com/2023/06/08/google-to-crack-down-on-hybrid-work-asks-remote-workers-to-reconsider.html), [shiny](https://gizmodo.com/apple-remote-work-return-to-office-hybrid-work-1850256210) [offices](https://www.cnn.com/2022/06/01/tech/elon-musk-tesla-ends-work-from-home/index.html), then chances are you have a two-tiered system."

Rather than saying "If you're a company..." maybe replace with just "Companies built around..."

> "But the policy actually makes sense for Dell. Sort of. When you’re a decades-old behemoth built around people working in offices, implementing remote work policies that actually work is hard. But it doesn't have to be. There are some key principles that any company – big or small, young or old, or Dell – can follow to realize the potential of remote working."

Reframe to make it more about "here's how to avoid becoming Dell" / "you don't have to build you company like this" etc. Might be cool to mention massive companies that have done remote/hybrid successfully. MongoDB is a famous one afaik.

Reply:
have hopefully addressed this in minor edits throughout